mailbox/api/src/test/java/org/apache/james/mailbox/events/GroupContract.java

```diff
@@ -275,12 +275,50 @@ default void allGroupListenersShouldBeExecutedWhenGenericGroups() throws Excepti
             verify(listener1, timeout(ONE_SECOND.toMillis()).times(1)).event(any());
             verify(listener2, timeout(ONE_SECOND.toMillis()).times(1)).event(any());
         }
+
+        @Test
+        default void groupListenerShouldReceiveEventWhenRedeliver() throws Exception {
```

Comment:
Can we have 2 registered group listener A & B then redeliver to B and ensure A did not receive it?